### PR TITLE
Field-Array-Implementation

### PR DESCRIPTION
**Pull request by alemanialloyd**
Converts arr+ and arr- with arrayUnion and arrayRemove respectively in onMessageInstructionsQueue.

### CODE REVIEW

File: src/utils/distribution.ts [hunk 1]
```diff
@@ -103,6 +103,27 @@ export async function onMessageInstructionsQueue(event: CloudEvent<MessagePublis
...
+      } else if (instruction.startsWith("arr+") || instruction.startsWith("arr-")) {
+        const regex = /\((.*?)\)/;
+        const match = instruction.match(regex);
+
+        if (!match) {
+          console.log(`Invalid instruction ${instruction} for property ${property}`);
+          continue;
+        }
+
+        const stringValues = match[1];
```
Comment (alemanialloyd) [inline src/utils/distribution.ts:115]: rename stringvalues to params

Comment (aris-i) [reply src/utils/distribution.ts]: maski pala paramsStr na lang pala ito

File: src/utils/distribution.ts [hunk 2]
```diff
@@ -103,6 +103,27 @@ export async function onMessageInstructionsQueue(event: CloudEvent<MessagePublis
...
+          continue;
+        }
+
+        const params = match[1];
+        if (!params) {
+          console.log(`No values found in instruction ${instruction} for property ${property}`);
+          continue;
+        }
+
+        const arrayValues = params.split(",").map((value) => value.trim());
```
Comment (aris-i) [inline src/utils/distribution.ts:121]: Tapos ito instead na arrayValues, mas maganda params name

**Review by aris-i — CHANGES_REQUESTED** [review]
May small changes lang Lloyd

**Review by aris-i — APPROVED** [review]
Nice

**Pull request merged**